src/wizard/boards.ts

```diff
@@ -1,22 +1,26 @@
 import { html } from "lit";
 import { SupportedBoards } from "../api/boards";
 
-export const boardSelectOptions = (boards: SupportedBoards | undefined) =>
+export const boardSelectOptions = (
```

Comment:
we can move this logic inline into the renderBoard step. It was initially here because it contained a huge hardcoded list.

Reply:
Method inlined.